files/www/cgi-bin/status

```diff
@@ -100,6 +100,20 @@ function get_default_gw()
     return "none"
 end
 
+function get_memavail()
+    local f = io.open("/proc/meminfo", "r")
+    if f then
+      for line in f:lines()
+      do
+        local memavail = line:match("^MemAvailable:%s+(%d+)%s+kB")
+        if memavail then
+          f:close()
+          return tonumber(memavail)
+        end
+      end
```

Comment:
Need to also close the file here - if somehow we didn't find MemAvailable we still need to close the file

Reply:
Good catch, @aanon4 Tim.  THX